src/sources/util/framestream.rs

```diff
@@ -157,11 +155,6 @@ impl FrameStreamReader {
         } else {
             //data frame
             if self.state.control_state == ControlState::ReadingData {
-                emit!(SocketEventsReceived {
```

Comment:
As far as I can see, the only source that calls this function is `dnstap`, so that is the only source that I have had to add emitting this event to since it can no longer be emitted here (we need to have created the event in order to know it's JsonSize).